client/src/components/App/App.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { FC, useEffect } from 'react';
 import { BrowserRouter as Router, Redirect, Route, Switch } from 'react-router-dom';
-import { StyledEngineProvider } from '@mui/material/styles';
+import StyledEngineProvider from '@mui/styled-engine/StyledEngineProvider';
```

Comment:
Is there documentation about this one? Everything I've been finding has been using `@mui/material/styles`

Reply:
I agree.  Will revert this to line up with the docs.  This was correct.  The bundle size thread was a red herring but caught the other imports.